tests/network/ping.pm

```diff
@@ -0,0 +1,41 @@
+# SUSE's openQA tests
+#
+# Copyright SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: iputils
+# Summary: Ping another machine
+# Maintainer: Dominik Heidler <dheidler@suse.de>
+
+use base 'consoletest';
+use testapi;
+use serial_terminal 'select_serial_terminal';
+use strict;
+use warnings;
+use utils;
+use lockapi;
+use mmapi 'wait_for_children';
+
+sub run {
+    if (get_var('IS_MM_SERVER')) {
+        barrier_create 'SETUP_DONE', 2;
+        mutex_create 'barrier_setup_done';
+    }
+
+    mutex_wait 'barrier_setup_done';
+
+    select_serial_terminal;
+    barrier_wait 'SETUP_DONE';
+
+    if (get_var('IS_MM_CLIENT')) {
+        my $remote = '10.0.2.101';
+        my $size = get_var('MM_MTU', 1380) - 28;    # 8 bytes ICMP and 20 bytes IPv4 header
+        script_retry("ping -M do -s $size -c1 -W1 $remote", retry => 5, delay => 2, fail_message => "Failed to ping $remote");
+    }
+
+    # Finish job
+    wait_for_children if (get_var('IS_MM_SERVER'));
+
```

Comment:
```suggestion
use constant WHATEVER_1380_MEANS => 1380;
use constant ICMP_IPV4_HEADER_SIZE => 28;  # 8 bytes ICMP and 20 bytes IPv4 header
…
    mutex_wait 'barrier_setup_done';
    select_serial_terminal;
    barrier_wait 'SETUP_DONE';
    if (get_var('IS_MM_CLIENT')) {
        my $remote = '10.0.2.101';
        my $size = get_var('MM_MTU', WHATEVER_1380_MEANS) - ICMP_IPV4_HEADER_SIZE;
        script_retry("ping -M do -s $size -c1 -W1 $remote", retry => 5, delay => 2, fail_message => "Failed to ping '$remote'");
    }
    # Finish job
    wait_for_children if get_var('IS_MM_SERVER');
```

and why the `script_retry`?

Reply:
> and why the script_retry?

Actually, why not? We want to know life is good, so not dying on the first ping is a good idea, perhaps 5 is a bit too much, but isn't the end of the world. 

> WHATEVER_1380_MEANS

I guess that's the MTU packet size

Reply:
I think the comment  should be descriptive enough.